tests/test_functional.py

```diff
@@ -7,6 +7,7 @@
 from fast_llm.utils import Assert
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA is not available")
```

Comment:
Any way to make a `@requires_gpu` fixture instead of copying this everywhere?

Reply:
done